src/hooks/useBreakpoint.tsx

```diff
@@ -0,0 +1,41 @@
+import { isEqual } from "lodash";
```

Comment:
Suggestion: @blimmer has done some performance/bundle optimizations for us, and it turns out these "lodash" imports pull in all of lodash and so end up being something we avoid.

Just doing:

```
import isEqual from "lodash/isEqual";
```

I believe is the kosher/tree-friendly way of importing this. 

(I also wouldn't mind using `import equal from "fast-deep-equal"`, which we use in `useComputed` so is already in Beam/being included in our bundle)

Reply:
ah yes, I swear I had (lodash/isEqual) at one point but must have been messing around and lost it but anyway -  switched to using `fast-deep-equal` as to prevent an additional need in the bundle..

Reply:
Great!